src/components/PaymentStripe/index.d.ts

```diff
@@ -0,0 +1,16 @@
+import { ReactNode, MouseEvent, FormEvent } from 'react';
+import { PaymentIntent, StripeError } from '@stripe/stripe-js';
+import { BaseProps } from '../types';
+
+export interface PaymentStripe extends BaseProps {
+    apiKeyGMap?: string;
```

Comment:
let's use a simple Input instead of GoogleAddressLookup and then remove this prop